press/api/client.py

```diff
@@ -299,6 +304,9 @@ def check_team_access(doctype: str, name: str):
 	if frappe.local.system_user():
 		return
 
+	if has_role("Press Support Agent") and doctype in ALLOWED_DOCTYPES_FOR_SUPPORT:
+		return
+
```

Comment:
this shouldn't be here? it changes the meaning of `check_team_access` function. Or rename the function to `check_document_access`